examples/nextjs-file-upload/README.md

```diff
@@ -0,0 +1,30 @@
+# Upload a file to Linear with Next.js
+
+This example shows how to upload a file to Linear using the Linear TypeScript SDK and Next.js API Routes. To run this example, you'll need a Linear account and a Linear API key. You can create an API key from your personal [Linear settings](https://linear.app/settings/api). Learn more about authentication in the [Linear Developer documentation](https://developers.linear.app/docs/sdk/getting-started#2.-create-a-linear-client).
```

Comment:
Should we be more explicit that while this example is in Next.JS the principals apply to any Node backend?